xbmc/GUIInfoManager.h

```diff
@@ -152,6 +152,7 @@ class CGUIInfoManager : public IMsgTargetCallback, public Observable,
   std::string GetDuration(TIME_FORMAT format = TIME_FORMAT_GUESS) const;
 
   void SetCurrentItem(CFileItem &item);
+  void SetCurrentItemInternal(CFileItem &item);
```

Comment:
It's called from JobManager so needs to be public.
Perhaps naming it SetCurrentItemBlocking or using the existing function with a blocking parameter would be better?